docs/how-to-guides/SPICE/using-web-spice-in-isis-and-ale.md

```diff
@@ -0,0 +1,31 @@
+# Using Web Spice in ISIS and ALE
+
+## ISIS
+
+Update your *IsisPreferences* under `$CONDA_PREFIX` to enable the web SpiceQL feature.
+
+```
+Group = WebSpice
+  UseWebSpice = "true"
+EndGroup
+```
+
+!!! tip "spiceinit"
+    The ISIS app *spiceinit* has two driver engines: ISIS and ALE. Setting `web=yes` will use the [old ISIS SPICE webserver](https://astrogeology.usgs.gov/docs/how-to-guides/environment-setup-and-maintenance/isis-data-area/#isis-spice-web-service) but you will still need to download the `base` directory to your `ISISDATA` path. If your *IsisPreferences* has the web SpiceQL feature enabled, *spiceinit* will use SpiceQL via the ALE engine to load kernels and no local data area will be necessary.
```

Comment:
as of right now, shape models are still required. So you download only shape models with `downloadIsisData <mission> --no-kernels $ISISDATA`